docs/userguide/main.md

```diff
@@ -1167,4 +1167,47 @@ spec:
         serverName: "testserver.test.local"
         insecureSkipVerify: true
   # ...
+
+### Managing ISM policies with Kubernetes resources
+
+The operator provides a custom Kubernetes resource that allow you to create/update/manage ISM policies using Kubernetes objects.
+
+#### OpenSearch ISM policy
```

Comment:
As long as the operator only manages one type of policy there is no need for the extra subheading